### ISSUE TITLE
stress: failed test in cockroach//util/interval/interval.test: TestRangeListAndRangeGroupIdentical

### ISSUE THREAD

**Issue by BramGruneir** [cockroachdb/cockroach#4841]
The following test appears to have failed:

```
=== RUN   TestRangeListAndRangeGroupIdentical
--- FAIL: TestRangeListAndRangeGroupIdentical (0.00s)
	range_group_test.go:544: expected subtracting [4e26f4-79d806) from RangeList [[1999ac-4e26f4) [c2cffa-e8d70e)] and RangeTree [[1999ac-4e26f4) [c2cffa-e8d70e)] to produce the same result; got false from RangeList and true from RangeTree
FAIL


ERROR: exit status 1
```
Please assign, take a look and update the issue accordingly.

**Comment by nvanbenschoten** [comment]
This issue was fixed in https://github.com/nvanbenschoten/cockroach/commit/8eebc805b4f659a64554fdeaf2990dd14f7df00f.

**Comment by nvanbenschoten** [comment]
Fixed in #4842.
